bazel/api_build_system.bzl

```diff
@@ -146,14 +146,25 @@ def udpa_proto_package(srcs = [], deps = [], has_services = False, visibility =
         ],
     )
 
-def udpa_cc_test(name, **kwargs):
+def xds_cc_test(name, **kwargs):
     native.cc_test(
         name = name,
         **kwargs
     )
 
-def udpa_go_test(name, **kwargs):
+def xds_go_test(name, **kwargs):
     go_test(
         name = name,
         **kwargs
     )
+
+# Old names for backward compatibility.
+# TODO(roth): Remove these once all callers are migrated to the new names.
```

Comment:
This will be removed in this PR?

Reply:
I was thinking that these build rules might be pulled in by projects that use this repo, so I didn't know if I could remove the old names without breaking anyone.  But if we know that no one is using these rules, then I'm happy to remove them immediately.